kubernetes/kubernetes-client-core.cabal

```diff
@@ -8,16 +8,16 @@ description:    .
                 .
                 base path: http://localhost
                 .
-                Kubernetes API version: v1.13.4
+                Kubernetes API version: v1.14.2
                 .
                 OpenAPI version: 3.0.1
                 .
 category:       Web
-homepage:       https://github.com/kubernetes-client/haskell
-author:         Auto Generated
-maintainer:     Shimin Guo <smguo2001@gmail.com>
-license:        Apache-2.0
-license-file:   LICENSE
+homepage:       https://openapi-generator.tech
+author:         Author Name Here
+maintainer:     author.name@email.com
+copyright:      YEAR - AUTHOR
+license:        UnspecifiedLicense
```

Comment:
Are you using the latest master version of https://github.com/kubernetes-client/gen? It should not have changed these fields.

Reply:
Yes, i'm using the latest version

Reply:
What was the command you used to generate the code?

We override these fields here: https://github.com/kubernetes-client/gen/blob/3e160e6416503e9983c0ff2ff4f05e4455bc393c/openapi/haskell.sh#L55-L69

Reply:
I'm using `./$(GEN_ROOT)/openapi/haskell.sh kubernetes settings`

Reply:
```
CABAL_OVERRIDES=(homepage https://github.com/kubernetes-client/haskell
           author "Auto Generated"
           maintainer "Shimin Guo <smguo2001@gmail.com>"
           license Apache-2.0)

patch_cabal_file() {
    while [[ $# -gt 1 ]]; do
        sed -i 's|^\('$1':[[:space:]]*\).*|\1'"$2"'|' ${OUTPUT_DIR}/*.cabal
        shift 2
    done
}
patch_cabal_file "${CABAL_OVERRIDES[@]}"

# Add license-file after license
sed -i '/^license:/a license-file:   LICENSE' ${OUTPUT_DIR}/*.cabal

ln -sf ../LICENSE ${OUTPUT_DIR}/LICENSE

sed -i '/^copyright:/d' ${OUTPUT_DIR}/*.cabal
echo "---Done."

```

Reply:
@guoshimin I found the error
```
sed: 1: "/Users/irvi.aini/dev/k8 ...": command i expects \ followed by text
```

```
uname -a
jessie 18.2.0 Darwin Kernel Version 18.2.0: Thu Dec 20 20:46:53 PST 2018; root:xnu-4903.241.1~1/RELEASE_X86_64 x86_64 i386 MacBookPro14,1 Darwin
```

Reply:
FIxed by changing `sed` with `gsed`